components/eventing-controller/reconciler/backend/reconciler_test.go

```diff
@@ -416,8 +416,8 @@ var _ = Describe("Backend Reconciliation Tests", func() {
 					EventingReady:               utils.BoolPtr(false),
 					SubscriptionControllerReady: utils.BoolPtr(false),
 					PublisherProxyReady:         utils.BoolPtr(false),
-					BebSecretName:               "",
-					BebSecretNamespace:          "",
+					BebSecretName:               bebSecret1name,
```

Comment:
There should not be any bebsecret in the status as reconciler failed to stop the NATS controller. I think empty should be the correct value.

Reply:
We are using BEB as backend and the name/namespace is what is expected. Not ready status is due to not stopping the nats subscription controller. I think this also makes sense.

Reply:
Right, I overlooked, bebsecret name/namespace do get updated.